client/src/profile/ProfileForm.js

```diff
@@ -1,19 +1,11 @@
 import React, { PropTypes } from 'react';
 import { reduxForm, Field } from 'redux-form';
 import { AuthField } from '../auth/AuthField';
-import authValidation from '../auth/authValidation';
+import profileValidation from './profileValidation';
 
 const ProfileForm = ({ handleSubmit, submitting, valid }) => (
   <form className="usa-form" onSubmit={handleSubmit}>
     <fieldset>
-      <legend className="usa-drop_text">Update your profile</legend>
```

Comment:
Legend is needed for accessibility purposes, although feel free to change the text